tests/braille-specs/rw-rw-g1.yaml

```diff
@@ -0,0 +1,4964 @@
+# Kinyarwanda Grade 1 Braille
+# Copyright (C) 2022 by Compass Braille <http://www.compassbraille.org>
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# ----------------------------------------------------------------------------------------------
+display: |
+  include unicode.dis
+  display 9 9
+  display a a
+table:
+  language: rw
+  region: rw
+  grade: 1
+  __assert-match: rw-rw-g1.utb
+
+flags: {testmode: bothDirections}
+tests:
+  - ['\x0009', '9']
+  - ['\x0020', '\x2800']
+  - ['\x00a0', 'a']
+flags: {testmode: forward}
+tests:
+  - ['\x000a', '\x2800']
+  - ['\x000b', '\x2800']
+  - ['\x000c', '\x2800']
+  - ['\x000d', '\x2800']
+  - ['\x2000', '\x2800']
+  - ['\x2001', '\x2800']
+  - ['\x2002', '\x2800']
+  - ['\x2003', '\x2800']
+  - ['\x2004', '\x2800']
+  - ['\x2005', '\x2800']
+  - ['\x2006', '\x2800']
+  - ['\x2007', '\x2800']
+  - ['\x2008', '\x2800']
+  - ['\x2009', '\x2800']
+  - ['\x200a', '\x2800']
+  - ['\x2028', '\x2800']
+  - ['\x2029', '\x2800']
+  - ['\x202f', '\x2800']
+  - ['\x205f', '\x2800']
+
+
+# List of words which currently only work in forward direction:
+flags: {testmode: forward}
+tests:
+  - ['18.39—19.16', '⠠⠁⠓⠲⠉⠊⠤⠤⠠⠁⠊⠲⠁⠋'] # backtranslated as: 18.39–19.16
+  - [')', '⠶'] # backtranslated as: (
+  - ['9.43b-45', '⠠⠊⠲⠙⠉⠨⠃⠤⠠⠙⠑'] # backtranslated as: 9.43B-45
+  - ['9.37-43a', '⠠⠊⠲⠉⠛⠤⠙⠉⠨⠁'] # backtranslated as: 9.37-43A
+
+
+###############################################################
+# List of words which work in both directions:
```

Comment:
Were these words actually proofread or is the list auto-generated? A comment would be good.

Reply:
Have added comment

Reply:
Thank you.